tests/cases.py

```diff
@@ -1684,6 +1684,17 @@ def test_process_tail_scores_(self) -> None:
             dense_positive_mask=mask,
         )
 
+    def test_process_relation_scores_(self) -> None:
+        """Test the evaluator's ``process_relation_scores_()`` function."""
+        hrt_batch, scores, mask = self._get_input()
+        true_scores = scores[torch.arange(0, hrt_batch.shape[0], device=hrt_batch.device), hrt_batch[:, 1]][:, None]
```

Comment:
@mberr can you look into why this is failing

Reply:
the generated scores are in shape `*, num_entities` instead of `*, num_relations`